components/EditionMenu.vue

```diff
@@ -78,6 +99,16 @@
             border-left: $border-width solid $light-grey-100;
         }
     }
+
+    .crossed {
+        &::after {
+            content: '/';
+            position: absolute;
+            color: $secondary;
+            transform: scaleX(-1) rotate(130deg) translate(-140%, -55%);
```

Comment:
![image](https://github.com/comixcraft/Editor/assets/85215120/1053c5cb-0154-4f4f-a913-67667528d22e)
As you can see on the image, the line is not centered on the element.

If you change the trasform for translate(-160%,-18%) rotate(10deg) it might be better.

I would also make it bold so that it's easier to see.

Reply:
Changed it to have different icons in the end